scripts/transactions_exporter.py

```diff
@@ -66,31 +76,30 @@ def get_token_transfers(token, start_block, end_block) -> pd.DataFrame:
         filter_by_name('Transfer', token.abi)[0],
         web3.codec,
     )
-    postgres.cache_token(token.address)
-    decimals = contract(token.address).decimals()
+    token_entity = cache_token(token.address)
     events = decode_logs(
         get_logs_asap(token.address, topics, from_block=start_block, to_block=end_block)
     )
     return pd.DataFrame(
         Parallel(1, 'threading')(
-            delayed(_process_transfer_event)(event, token, decimals) for event in events
+            delayed(_process_transfer_event)(event, token_entity) for event in events
         )
     )
 
 
-def _process_transfer_event(event, vault, decimals) -> dict:
+def _process_transfer_event(event, token_entity) -> dict:
     sender, receiver, amount = event.values()
-    postgres.cache_address(sender)
-    postgres.cache_address(receiver)
-    price = _get_price(event, vault)
+    cache_address(sender)
+    cache_address(receiver)
+    price = _get_price(event, token_entity)
     if (
-        vault.address == '0x7F83935EcFe4729c4Ea592Ab2bC1A32588409797'
+        token_entity.token.address == '0x7F83935EcFe4729c4Ea592Ab2bC1A32588409797'
```

Comment:
maybe add a comment what that is and extract as constant

Reply:
I had a comment below the if statement but now I've moved it inside instead